### ISSUE TITLE
CSS Rule `color` & `border-color` missed from `.app-item i` selector in the new release 1.7.10

### ISSUE THREAD

**Issue by fasih** [MaistrenkoAnton/django-material-admin#125]
I have dynamically loaded ` var(--main-bg-color)` for different user accounts to provide UI customisation.
Now app icon color is missing due to below SCSS Rule

```
 .collapsible-body span.badge, .app-list .app .card .app-item i, .card .card-content {
   .card-app-name {
     width: 90%;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
     margin-left: auto;
     margin-right: auto;
   }

   .card-title {
     color: var(--main-bg-color);
     border-color: var(--main-bg-color);
   }
 }
```

Initially it was

```
.collapsible-body span.badge, .app-list .app .card .app-item i, .card .card-content .card-title {
  color: var(--main-bg-color);
  border-color: var(--main-bg-color);
}
```

**Comment by MaistrenkoAnton** [comment]
Thanks, fixed in v`1.7.11`

**Comment by fasih** [comment]
> Thanks, fixed in v`1.7.11`

💯 Lightning fast. Thanks for the quick turnaround time. I am closing this issue
